src/Elocrypt.php

```diff
@@ -60,14 +61,20 @@
  */
 trait Elocrypt
 {
-    /** @var string The prefix used to determine if a string is encrypted. */
-    protected static $ELOCRYPT_PREFIX = '__ELOCRYPT__:';
-
     //
     // Methods below here are native to the trait.
     //
 
     /**
+     * Get the configuration setting for the prefix used to determine if a strign is encrypted
+     *
+     * @return string
+     */
+    protected function getElocryptPrefix() {
+        return Config::get('elocrypt.prefix');
```

Comment:
This is non-BC if someone has not merged in the config file.
A better approach would be:

```php
if (Config::has('elocrypt.prefix') {
    return Config::get('elocrypt.prefix');
}
return '__ELOCRYPT__:';